### DISCUSSION TITLE
Modern logo request

### DISCUSSION THREAD

**Discussion by FlavioAdamo** [Exifly/ApiVault#103] [Show and tell]
## Is your feature request related to a problem? Please describe.
The current logo for the website is outdated . It does not match the minimalistic design of the rest of the website and needs to be updated to provide a more modern look and feel.

## Describe the solution you'd like
We would like a new, minimalistic logo that matches the design of the website. The logo should be simple and clean, yet visually appealing and easy to recognize. It should be easy to reproduce in different sizes and on different mediums, such as on social media platforms and printed materials.

## Describe alternatives you've considered
We have considered keeping the existing logo and making minor adjustments, but we believe that a complete redesign is necessary to achieve the desired look and feel.

**Comment by NirajD10** [comment]
For me I think the current design is good one so it shouldn't change logo. But in case let consider what people does think of it.

**Comment by imggion** [comment]
Ok, we'll converting this issue in a Discussion to talk about the logo, thanks!
